app/scripts/metamask-controller.js

```diff
@@ -81,7 +84,7 @@ module.exports = class MetamaskController {
     const noticeController = this.noticeController
 
     return {
-      getState: (cb) => { cb(null, this.getState()) },
+      getState: nodeify(this.getState.bind(this)),
```

Comment:
I just think nodeify is so canonical, that I actually think more readers will be confused trying to interpret some new "semantic" wording than if we leave it.  Nodeify/denodeify are a very well defined set of functions.